css/mobile.scss

```diff
@@ -50,11 +49,6 @@
 		padding-right: 12px;
 	}
 
-	#mail-content,
-	.mail-message-attachments {
-		margin: 0 10px 50px 30px;
-	}
```

Comment:
Any specific reason this was removed? This seems to cause the issue with the content being moved to the left. :)

Reply:
I think he wanted to use all the available space which could be more important then aligning all content at the same border...
What about the header of the message: couldn't it be always left-aligned and move to the right if the mobile view is activated to free the spacw for the back-button?

Reply:
@szaimen let’s please keep this pull request on topic. Moving the text so it sticks to the very left of the content is certainly a bug and not intentional:
![image](https://user-images.githubusercontent.com/925062/77706540-f69c5300-6fc2-11ea-9b24-21ed41faf4f8.png)

Reply:
Fixed

Reply:
this wasn't really intentional, sorry... wanted to remove the margin from the attachment section but not from the body/content, obviously...
thanks for fixing this! @jancborchardt